gatsby-site/src/components/team/team.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import TeamMember from '../team-member/team-member'
+import styles from "./team.css"
+const alina = "https://res.cloudinary.com/colark/image/upload/v1534305485/alina-photo.png"
+const ellie = "https://res.cloudinary.com/colark/image/upload/v1534305486/ellie-photo.png"
+const lian = "https://res.cloudinary.com/colark/image/upload/v1534305485/lian-photo.png"
+const olga = "https://res.cloudinary.com/colark/image/upload/v1534305498/olga-photo.png"
+const claire = "https://res.cloudinary.com/colark/image/upload/v1534456735/Colark%20Marketing%20Site/claire-photo.jpg";
+
+function Team(props) {
+  const teamMembers = [{
+      name: "Ellie Day",
+      className: "ellie-photo team-member__photo-div",
+      position: "Founder, Engineer",
+      photo: ellie,
+      email: "ellie@colark.com",
+      linkedin: "https://www.linkedin.com/in/ellie-day-a7761394/"
+    },
+    {
+      name: "Olga Kuri",
+      className: "olga-photo team-member__photo-div",
+      position: "Visual Designer/ Illustrator",
+      photo: olga,
+      email: "olga@colark.com",
+      linkedin: "https://www.linkedin.com/in/kuriolga/"
+    },
+    {
+      name: "Alina Lodahl",
+      className: "alina-photo team-member__photo-div",
+      position: "Software Engineer",
+      photo: alina,
+      email: "alina@colark.com",
+      linkedin: "https://www.linkedin.com/in/alina-lodahl/"
+    },
+    {
+      name: "Lian Thompson",
+      className: "lian-photo team-member__photo-div",
+      position: "Software Engineer",
+      photo: lian,
+      email: "lian@colark.com",
+      linkedin: "https://www.linkedin.com/in/lianthompson/"
+    },
+    {
+      name: "Claire Gallant",
+      className: "claire-photo team-member__photo-div",
+      position: "Product Manager",
+      photo: claire,
+      email: "claire@colark.com",
+      linkedin: "https://www.linkedin.com/in/claire-gallant-4494b683/"
+    },
+    {
+      name: "Someone",
+      className: "someone-photo team-member__photo-div",
+      position: "Software Engineer",
+      photo: "",
+      email: "someone@colark.com",
+      linkedin: ""
+    }
+  ];
+  let team = teamMembers.map((member, index) => {
+    return(
+    <div className="team-member-div" key={ index }>
+      <TeamMember data={ member } />
```

Comment:
Thoughts on mapping member directly onto TeamMember props like `<TeamMember 
 {...member} />`? You wouldn't need the data key then. Or was that intentional?